Exiled.API/Features/Player.cs

```diff
@@ -252,6 +252,11 @@ public string CustomPlayerInfo
         /// </summary>
         public bool DoNotTrack => ReferenceHub.serverRoles.DoNotTrack;
 
+        /// <summary>
+        /// Gets a value indicating whether the player is connected to the server or not.
+        /// </summary>
+        public bool IsConnected => List.Contains(this);
```

Comment:
Bad implementation, when a client is disconnected, Mirror destroys its GameObject (destroying a GameObject includes its components), the simple thing you must do is check a component for nullability, it can be done with ReferenceHub itself